tests/integration/rooms.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { setup, clearDb, get, post } from '../utils';
+import { getRepository } from 'typeorm';
+import * as HttpStatus from 'http-status-codes';
+
+import { Room } from '../../src/entity/Room';
+
+describe('Rooms Routes', () => {
+  beforeAll(setup);
+  beforeEach(clearDb);
+
+  describe('GET /rooms/:id', () => {
+    it('should return a room', async () => {
+      const room = await getRepository(Room).save({
+        name: 'foo',
+        description: 'bar',
+        options: [{ label: 'option label', value: 'option value' }],
+      });
+      const res = await get(`/rooms/${room.id}`);
+      expect(res.status).toEqual(HttpStatus.OK);
+      expect(res.body).toHaveProperty('data', { ...room });
+    });
+
+    it("should a 404 error if a room doesn't exists", async () => {
+      const res = await get(`/rooms/non-existent-id`);
+      expect(res.status).toEqual(HttpStatus.NOT_FOUND);
+    });
+  });
+
+  describe('POST /rooms', () => {
+    it('should create a room', async () => {
+      const room = {
+        name: 'foo',
+        description: 'bar',
+        options: [{ label: 'option label', value: 'option value' }],
+      };
```

Comment:
Since we are reusing this same object between tests I would declare it as a top level variable.

Reply:
Yes, I can do that. I wanted to replace these object declarations with factories but I guess we can do it later

Reply:
tell me if you agree with the new changes